<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NListaDialog</class>
 <widget class="QDialog" name="NListaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 241, 244)</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>20</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="NListaInvalido">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="NLDialogNome">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Nome da lista:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="NomeEnt">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>21</y>
     <width>136</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnNListaTarefas">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(5, 68, 94);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="TextoNListaTarefas">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lista de Tarefas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>131</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnNListaCompras">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(5, 68, 94);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="TextoNListaCompras">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lista de compras</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
